src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,22 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    private static final int MAX_NUMBER = 10;
+    private static final int MIN_NUMBER = 1;
+    private final Random random = new Random();
+
+    @Override
+    public List<Excavator> get() {
+        int number = random.nextInt(MAX_NUMBER) + MIN_NUMBER;
```

Comment:
there is no sense to use two numbers (min and max)

you could do (max number + 1)

however do you really need `Random` here? if  I were you I would simply use my max number in `for` loop :)